srcpkgs/ly/files/ly/run

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+[ -r conf ] && . ./conf
+
+chvt "${TTY}"
```

Comment:
This is terrible. If the service is constantly dying for some reason, you can't fix it at all, because it will always run `chvt` again.

Reply:
I just tested it again without the `chvt` and it works just fine; I think I might have also changed something else last time. I've removed the `chvt` and `reset` because they're no longer needed.